### PR TITLE
Changes as per ARB except Report Format and DOC added 

### PR DESCRIPTION
**Pull request by cs-rushikesh-mehakare**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,30 @@
-# solution-template-repository
\ No newline at end of file
+# Release Information
+
+ * **Version**: 1.0.0
+ * **Certified**: No
+ * **Publisher**: Fortinet CSE
```
Comment (cs-ankit-gupta) [inline README.md]: Mark it as Fortinet

File: README.md [hunk 2]
```diff
@@ -1 +1,30 @@
...
+ * **Compatible Version**: 
+   * FortiSOAR v7.4.3 and later
```
Comment (cs-ankit-gupta) [inline README.md]: Make it in single line

File: README.md [hunk 3]
```diff
@@ -1 +1,30 @@
...
+ * [Release Notes](./release_notes.md)
```
Comment (cs-ankit-gupta) [inline README.md]: remove release not as it is first version

File: docs/contents.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+# Contents
+
+The **System Configuration Recommendation** solution pack contains the following resources.
+
+## Connectors
+
+|**Name**|**Description**|
+| :- | :- |
+| SSH  |  _Pending_ Use this connector to connect to different servers and execute commands or python scripts on those servers. |
```
Comment (cs-ankit-gupta) [inline docs/contents.md]: remove _Pending_ it not required

File: docs/contents.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+
+
+## Record Set
+
+|**Name**|**Description**|
+| :- | :- |
+|  Scenario  |  Record named as **System Configuration Recommendation** is used to trigger the playbooks.  |
```
Comment (cs-ankit-gupta) [inline docs/contents.md]: Re describe the description

File: docs/contents.md [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+
+## Playbook Collection
+
+|**Playbook Collection Name**|**Description**|
+| :- | :- |
+| System Configuration Recommendation | Focuses on analysing and creating recommendation report |
```
Comment (cs-ankit-gupta) [inline docs/contents.md]: Add Playbook name and description used under the collection

File: docs/setup.md [hunk 7]
```diff
@@ -0,0 +1,18 @@
+| [Home](../README.md) |
+|--------------------------------------------|
+
+# Installation
```
Comment (cs-ankit-gupta) [inline docs/setup.md:3]: Make it as per standard refer to https://github.com/fortinet-fortisoar/solution-pack-phishing-email-response/blob/develop/docs/setup.md

File: docs/usage.md [hunk 8]
```diff
@@ -0,0 +1,25 @@
+| [Home](../README.md) |
+|--------------------------------------------|
+
+# Usage
+
+The `System Configuration Recommendation` Solution Pack is used to optimise FortiSOAR system. First, refer to the [installation and setup guide](setup.md) guide to get
```
Comment (cs-ankit-gupta) [inline docs/usage.md]: incomplete statment

File: info.json [hunk 9]
```diff
@@ -39,7 +39,7 @@
     "recordTags": [
         "System Configuration Recommendation"
     ],
-    "date": "2023-12-12T06:42:14+00:00",
+    "date": "2024-01-02T09:07:03+00:00",
     "contents": {
         "playbooks": [
             {
```
Comment (cs-ankit-gupta) [inline info.json:45]: Change Playbook collection name as 10 - SP - System Configuration Recommendation

File: records/scenario/scenario0001.json [hunk 10]
```diff
@@ -3,7 +3,7 @@
...
+        "description": "### This Scenario demonstrates configuring FortiSOAR based on your system details. \nBased on your system details, we suggest configuration changes for following \n\n1. PostgreSQL\n2. NGNIX\n3. UWSGI\n4. Celery-D \n\nYou can find all the recommendations in the attachment section inside FortiSOAR. \n\n* * *\n### How to use this SP \n\n1. Install SSH Connector \n2. Configure the above Connector for FortiSOAR VM\n3. And Simulate \"System Configuration Recommendation\" scenario\n4. After playbook's successful run you will get a Attachment Created by name \"Recommended_System_Configuration.html\"\n5. Apply the configurations as instructed in Recommended_System_Configuration report\n\n***\n\n### Note \n\n1. Make sure to install SSH connector and Configure it \n2. Some configuration only applies to system having SSD as storage\n3. One Task is also Created for **csadmin** user \n\n",
```
Comment (cs-ankit-gupta) [inline records/scenario/scenario0001.json]: Add suggested comment

File: playbooks/System Configuration Recommendation/Manual Input > System Configuration Recommendation.json [hunk 11]
```diff
@@ -0,0 +1,524 @@
...
+            "uuid": "f837c0b4-b25a-4132-a4f7-cf14829e5867",
+            "recordTags": []
+        }
+    ],
+    "priority": "\/api\/3\/picklists\/2b563c61-ae2c-41c0-a85a-c9709585e3f2",
+    "uuid": "a2f6f470-621f-44ec-99ad-3f30882e3d98",
+    "owners": [],
+    "isPrivate": false,
+    "deletedAt": null,
+    "recordTags": [
```
Comment (cs-ankit-gupta) [inline playbooks/System Configuration Recommendation/Manual Input > System Configuration Recommendation.json]: Add tags as per trigger step

File: playbooks/System Configuration Recommendation/System Configuration Recommendation.json [hunk 12]
```diff
@@ -607,7 +628,7 @@
             "left": "980",
             "height": "167",
             "width": "688",
-            "uuid": "cc077b07-5744-43ea-9ae9-7460b3e661d0",
+            "uuid": "d045e020-f8a5-4753-878a-2c10a4127511",
             "recordTags": []
```
Comment (cs-ankit-gupta) [inline playbooks/System Configuration Recommendation/System Configuration Recommendation.json:632]: Add recommended tags

File: playbooks/System Configuration Recommendation/System Configuration Recommendation.json [hunk 13]
```diff
@@ -146,7 +147,7 @@
                     "thread": false,
                     "content": "<h4><span style=\"color: rgb(224, 62, 45);\">Error : {% if (vars.system_config[\"error\"]| regex_search(\"PGSQL\") and (vars.system_config[\"error\"]| regex_search(\"PGSQL\")).strip())%}Please mount a drive for PGSQL directory{% elif (vars.system_config[\"error\"]| regex_search(\"command failed\") and (vars.system_config[\"error\"]| regex_search(\"command failed\")).strip())%}Command Failed{% elif (vars.system_config[\"error\"]| regex_search(\"following commands are installed or not\") and (vars.system_config[\"error\"]| regex_search(\"following commands are installed or not\")).strip())%}Make sure that all commands required are installed{% else%}{{vars}}{% endif %}<\/span><\/h4>\n<hr>\n<p>&nbsp;{{vars.system_config[\"error\"]}}<br>{{vars.system_config[\"error_log\"] if vars.system_config[\"error_log\"] is defined}}<\/p>",
                     "records": "{{vars.steps.Create_Task_For_Tracking_Purposes['@id']}}",
-                    "parentstepid": "\/api\/3\/workflow_steps\/2b6cab4e-6196-4710-84b2-ef7cc266950d"
+                    "parentstepid": "\/api\/3\/workflow_steps\/undefined"
```
Comment (cs-rohit-khune) [inline playbooks/System Configuration Recommendation/System Configuration Recommendation.json]: The parent step id should not be undefined, please check this step once

File: README.md (hunk 1 above)
Comment (cs-ankit-gupta) [reply README.md]: @cs-rushikesh-mehakare Not implemented

**Review by cs-ankit-gupta — APPROVED** [review]

**Pull request merged**